packages/fxa-content-server/app/scripts/templates/pair/index.mustache

```diff
@@ -1,21 +1,19 @@
 <div id="main-content" class="card">
   <header>
-    <h1 id="pair-header">{{#t}}Connect another device{{/t}}</h1>
+    <h2 id="pair-header">{{#t}}Get Firefox on your phone or tablet{{/t}}</h2>
   </header>
 
   <section>
     <div class="error"></div>
 
-    <div class="graphic {{ graphicId }}" role="img" aria-label="{{#t}}Scan to connect another device{{/t}}"></div>
-    <p>{{#t}}Open the code scanner on your Firefox app and show code when ready.{{/t}}</p>
+    <div class="graphic graphic-connect-another-device-qr-code" role="img" aria-label="{{#t}}Scan to connect another device{{/t}}"></div>
```

Comment:
I just wanted to double check that the aria-label doesn't also need to be updated. Previously the pair header said 'connect another device', but it no longer does. Perhaps this label should be inline with the header text, e.g. _Scan to get Firefox on your phone or table_.

Reply:
Great idea -- I wanted to follow up on this and forgot!